src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,196 @@
+        }
+        for (Car car : cars) {
+            car.setDrivers(getDriversForCar(driverId));
+        }
+        return cars;
+    }
+
+    private Car getCar(ResultSet resultSet) throws SQLException {
+        Long id = resultSet.getObject("cars.id", Long.class);
+        String model = resultSet.getString("model");
+        Long manufacturerId = resultSet.getLong("manufacturers.id");
+        Manufacturer manufacturer = new Manufacturer(manufacturerId,
+                resultSet.getString("name"), resultSet.getString("country"));
+        return new Car(id, model, manufacturer);
+    }
+
+    private void addDriversToCar(Car car) {
+        String query = "INSERT INTO cars_drivers (car_id, driver_id) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, car.getId());
+            for (Driver driver : car.getDrivers()) {
+                statement.setLong(2, driver.getId());
+                statement.executeUpdate();
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't add drivers to car: " + car + ".", e);
+        }
+    }
+
+    private void removeDriverFromCar(Long carId) {
+        String query = "DELETE FROM cars_drivers WHERE car_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, carId);
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't remove driver from car " + carId, e);
+        }
+    }
+
+    private List<Driver> getDriversForCar(Long id) {
+        String query = "SELECT * FROM drivers JOIN cars_drivers "
+                + "ON drivers.id = cars_drivers.driver_id "
+                + "WHERE cars_drivers.car_id = ? AND drivers.is_deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            List<Driver> drivers = new ArrayList<>();
+            while (resultSet.next()) {
+                drivers.add(getDriver(resultSet));
+            }
+            return drivers;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get drivers by car id: " + id, e);
+        }
+    }
+
+    private Driver getDriver(ResultSet resultSet) throws SQLException {
+        Long driverId = resultSet.getObject("drivers.id", Long.class);
+        String name = resultSet.getString("drivers.name");
+        String licenseNumber = resultSet.getString("drivers.licenseNumber");
```

Comment:
<img width="407" alt="Screenshot 2023-02-05 at 23 07 08" src="https://user-images.githubusercontent.com/103888898/216845926-70373912-06ba-44ad-a689-231a303ca1cc.png">
you do not have to change the license_number column name in the table